111.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 
 # lalala
-def printer(i=3):
+ def printer(i=3):
```

Comment:
E111 indentation is not a multiple of four
E113 unexpected indentation
E999 IndentationError: unexpected indent